pom.xml

```diff
@@ -86,7 +86,7 @@ THE SOFTWARE.
     <changelog.url>https://www.jenkins.io/changelog</changelog.url>
 
     <!-- Bundled Remoting version -->
-    <remoting.version>3148.v532a_7e715ee3</remoting.version>
+    <remoting.version>3160.vd76b_9ddd10cc</remoting.version>
```

Comment:
FTR https://github.com/jenkinsci/remoting/compare/3148.v532a_7e715ee3...3160.vd76b_9ddd10cc = https://github.com/jenkinsci/remoting/pull/675 plus some unrelated changes; see discussion in https://issues.jenkins.io/browse/JENKINS-72163

Reply:
Thanks for pointing to that difference!  I had missed that there is an additional bug fix that is included in remoting 3160 as delivered in Jenkins 2.428.  The additional fix is [JENKINS-65368 Remoting agent.jar does not work behind proxy](https://issues.jenkins.io/browse/JENKINS-65368).  I've labeled that bug report as `2.426.1-fixed` in Jira and included it in the pull request description